conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
A big part of the rational is tightly controlling imports.  There are READMEs in each of the subfolders, that are duplicated in the docstrings of the subfolders' `__init__`s.